src/app/effects/backend.wired.effects.ts

```diff
@@ -1,34 +1,34 @@
-import { Injectable, NgZone } from '@angular/core';
+import { Injectable } from '@angular/core';
 import { BlocklyEditorState } from '../state/blockly-editor.state';
 import { filter, withLatestFrom } from 'rxjs/operators';
 import { BackEndState } from '../state/backend.state';
 import { SketchStatus } from '../domain/sketch.status';
-import { BackEndMessage } from '../domain/backend.message';
 import { ConnectionStatus } from '../domain/connection.status';
 import { AppState } from '../state/app.state';
-import { RobotWiredState } from '../state/robot.wired.state';
 import { WorkspaceStatus } from '../domain/workspace.status';
-import { DialogState } from '../state/dialog.state';
 import { CodeEditorType } from '../domain/code-editor.type';
 import { NameFileDialog } from "../modules/core/dialogs/name-file/name-file.dialog";
 import { MatDialog } from "@angular/material/dialog";
 import { VariableDialog } from "../modules/core/dialogs/variable/variable.dialog";
 import { UploadDialog } from "../modules/core/dialogs/upload/upload.dialog";
 import { Router } from "@angular/router";
-import { CodeEditorState } from "../state/code-editor.state";
 import { DebugInformationDialog } from "../modules/core/dialogs/debug-information/debug-information.dialog";
 import * as Blockly from 'blockly/core';
+import {ConnectPythonDialog} from "../modules/core/dialogs/connect-python/connect-python.dialog";
+import {FileExplorerDialog} from "../modules/core/dialogs/file-explorer/file-explorer.dialog";
+import {CodeEditorState} from "../state/code-editor.state";
+import {PythonUploaderService} from "../services/python-uploader/PythonUploader.service";
 
 
 const fileExtensions = [
-  ".l_flitz_uno",
-  ".l_flitz_nano",
-  ".l_original_uno",
-  ".l_click",
-  ".l_uno",
-  ".l_nano",
-  ".l_wifi",
-  ".ino",
+	".l_flitz",
```

Comment:
dit klopt niet, moet l_flitz_uno zijn (dit was een merge conflict). Is onderdeel van de aanpassingen van de toolbox en zit op meer plekken